e2e/playwright/tests/smoke-test/test.spec.ts

```diff
@@ -61,6 +61,10 @@ test('smoke test', async ({ page }) => {
   await page.getByRole('link', { name: 'Config', exact: true }).click();
   await expect(page.locator('h3')).toContainText('My Example Config');
   await expect(page.locator('#version_sequence-group')).toContainText('This version is 1');
+  await expect(page.getByRole("combobox")).toBeVisible();
+  await page.getByRole("combobox").selectOption("option_1");
```

Comment:
nit: since option_1 seems to be the default, how about selecting option_2? same for the radio button, let's select option_2 and validate that it changes correctly.